src/codesearch.cc

```diff
@@ -439,10 +444,24 @@ code_searcher::~code_searcher() {
     }
 }
 
+void code_searcher::chunk_files() {
```

Comment:
Heh! This went away in the later commit where I started using a proper index (renamed to `index_filenames` or something), so I'll see about rebasing this away.